### ISSUE TITLE
When installing wp-graphql-gutenberg plugin, it gives error 500.

### ISSUE THREAD

**Issue by metinjakupi** [pristas-peter/wp-graphql-gutenberg#151]
GraphQL API for WordPress Version 1.8.3 

WP GraphQL Gutenberg  Version 0.3.11

**Comment by metinjakupi** [comment]
'WPGraphQLGutenberg\Schema\Types\Connection\Blocks\CoreImageBlockToMediaItemConnection' not found in  plugins/wp-graphql-gutenberg-0.3.12/src/Schema/Schema.php:12

**Comment by metinjakupi** [comment]
composer dump-autoload fixed the issue

**Comment by EarthlingDavey** [comment]
Hey I found you comment, unfortunately this didn't work for me. 
I did find a solution that might work more generally. 

It looks like this [commit](https://github.com/pristas-peter/wp-graphql-gutenberg/commit/4baec5b0c35843640e4c8b3ff20e893110a57721) fixes the issue.
And, it has not yet been released, but downloading from [https://github.com/pristas-peter/wp-graphql-gutenberg/archive/4baec5b0c35843640e4c8b3ff20e893110a57721.zip](https://github.com/pristas-peter/wp-graphql-gutenberg/archive/4baec5b0c35843640e4c8b3ff20e893110a57721.zip) worked for me.
